docs/src/main/sphinx/release/release-417.md

```diff
@@ -0,0 +1,40 @@
+# Release 417 (10 May 2023)
+
+## General
+
+* Improve performance of `UNION ALL` queries. ({issue}`17265`)
+* Fix over-reading data in the `TEXTFILE` decoder for structs that contain more
+  columns in the file than are mapped in the schema. ({issue}`17364`)
+
+## Delta Lake connector
+
+* Add support for [`COMMENT ON VIEW`](/sql/comment). ({issue}`17089`)
+* Improve efficiency of packing data in files written by the Parquet writer. ({issue}`17373`)
```

Comment:
This doesn't read well. How about something like "Write Parquet files more efficiently", or even "Improve performance when reading Parquet data written by Trino". cc @sopel39 @raunaqmorarka